ssc/cmod_battery_stateful.cpp

```diff
@@ -244,6 +245,7 @@ void write_battery_state(const battery_state& state, var_table* vt) {
     if (choice == lifetime_params::CALCYC) {
         vt->assign_match_case("q_relative_calendar", lifetime->calendar->q_relative_calendar);
         vt->assign_match_case("dq_relative_calendar_old", lifetime->calendar->dq_relative_calendar_old);
+        vt->assign_match_case("cycle_counts", lifetime->cycle->cycle_counts);
```

Comment:
since these are matrices, they'll have to follow the protection against discrepancy between empty arrays and matrix_t always having a num value-- like in line 236

Reply:
Nevermind, this matrix is never empty so this code and the one below are fine as is